specification/storage/resource-manager/Microsoft.Storage/stable/2019-06-01/examples/StorageAccountUpdate.json

```diff
@@ -49,7 +49,8 @@
             "bypass": "AzureServices",
             "defaultAction": "Allow",
             "ipRules": [],
-            "virtualNetworkRules": []
+            "virtualNetworkRules": [],
+            "resourceAccessRules": []
```

Comment:
Please change at least one example to use valid resourceAccessrule, instead of using empty rule in all examples. 
The example will be used in rest doc and swagger validate, without a valid rule in example, we can't validate the swagger change is right, and can't show customer how this value should be set in rest API.

Reply:
I will add the example in a different PR.

Reply:
Would suggest add sample in same PR, since:
1. The sample can be used to validate the swagger change is correct.
2. The doc will show customer how to set it, or customer don't have idea how to use it, even it's public in swagger.

Is there any specific reason need add it in another PR?